internal/provider/maas/environ.go

```diff
@@ -60,10 +60,7 @@ var defaultLongRetryStrategy = retry.CallArgs{
 	MaxDuration: 1200 * time.Second,
 }
 
-var (
-	DeploymentStatusCall = deploymentStatusCall
-	GetMAASController    = getMAASController
-)
+var GetMAASController = getMAASController
```

Comment:
This should have had a comment.

Reply:
Sure, this is a merge PR. Happy to fix it in another.